src/stdlib2.app.src

```diff
@@ -1,6 +1,6 @@
 {application, stdlib2, [ {description,  "stdlib2"}
                        , {vsn,          git}
                        , {registered,   []}
-                       , {applications, [kernel, stdlib]}
+                       , {applications, [kernel, stdlib, compiler]}
```

Comment:
What is the implication of this change for any library user? They will all get the compiler application now in their releases?

Reply:
Yes, but stdlib depends on compiler so they will already have it, but stdlib2 depends directly on compiler so this is dependency should be there. It is also necessary for dialyzer to pass.